### DISCUSSION TITLE
Logbook - Filter by area

### DISCUSSION THREAD

**Discussion by niemyjski** [home-assistant/frontend#7825] [Dashboard]
I'd love to filter logbook by an area. This would be extremely helpful when you have thousands of sensors constantly updating and you are wanting to see activity by area.

**Comment by KTibow** [comment]
You can already exclude sensors from logbook that you'd never want to look at, BTW.

**Reply by niemyjski** [reply]
That's not what I'm asking for or trying to do. I want to look at all of my entities in a given area. I wouldn't change 1800 entities to exclude just so I could all see the ones in one area.

**Reply by KTibow** [reply]
Well you can already exclude by default instead of include by default, it helps your HA storage live longer too. Not a solution, just a little workaround in the meantime.

If this was added, do you mean the area registry? Not all entities can be added to it, unless you use something like hass-magic-areas instead.

**Reply by niemyjski** [reply]
Yes, a defined area registry. I wish we could get motion sensors added to an area too as this would make this even more useful but baby steps. Thanks for pointing out that add-on!

**Reply by KTibow** [reply]
Does this mockup look good?
![ourliltreehouse duckdns org_config_dashboard](https://user-images.githubusercontent.com/10727862/101503510-09736f80-3927-11eb-93c8-2aa32fad6bfd.png)


> motion sensors added to an area

What do you mean?
If something is part of a device HA 1.0.0 added support for having 1 device's entities be in different rooms.

**Reply by niemyjski** [reply]
Yeah! I had some questions / feedback.

1. I'm wondering if it would be nice to have a label or tag under say Upstairs that says it's an area. 
2. If you hover over the drop down item would it list all the entities in that area? 
3. Would a user question as to what's included in an area as some sensors can't be in an area? This is in reference to your last question (`What do you mean?`).
4. Will the filter ever support multiple areas, entities (kinda weird specifying multiple with a drop down currently)?

**Reply by spacegaier** [reply]
The newly introduced "Selectors" might come in handy here to filter.

**Reply by KTibow** [reply]
> Selectors

Could you provide a link/screenshot?

**Reply by KTibow** [reply]
@niemyjski 
1. I considered that in the mock-up, but it looks odd given that everything else is an entity ID. If you added entities for areas (`area.living_room`) then it would look better IMO.
2. Just a mock-up. Feel free to compile a list of specs for anybody who wants to make an implementation.
3. Areas are defined by users when they add a device. I think most users know that only HA devices can be added to areas. Maybe there could be a tooltip or something?
4. Feel free to compile a mock-up. I agree check boxes or something would be better, but IMO the drop-down is crowded already.
5. If someone adds areas, it would make sense to add groups as well.

**Reply by spacegaier** [reply]
> 
> 
> > Selectors
> 
> Could you provide a link/screenshot?

E.g. here (the colored elements):
![](https://cdn.discordapp.com/attachments/427516175237382144/784480741642993694/unknown.png)

They are quite new (used to select elements for the new blueprints).

**Reply by KTibow** [reply]
I agree, selectors would look nice in the Logbook.
